### DISCUSSION TITLE
Could not update to latest version

### DISCUSSION THREAD

**Discussion by Lando021** [dbeaver/dbeaver#18763] [Q&A]
I need some help. 

I was trying to update DBeaver today but when I tried it failed. After it failed this error comes up. 
![image](https://user-images.githubusercontent.com/101134075/157089081-17a731ca-0309-4562-a3cd-ef85ac7f51f5.png)

I tried installing Java, but then another error kept coming up. 

I'm not a programer, but any help to avoid having to reinstall the whole app would be helpful. 

Thanks!

**Comment by serge-rider** [comment]
What Windows version do you use? (it has to be 64bit Windows).
How did you install DBeaver? (for All Users or for Current user).

If you are on 64bit Windows 7 or later then try to uninstall DBeaver and install it again.

**Comment by andrelinslima** [comment]
I would assume you installed dbeaver only for the current user, since the installation seems to be made inside the current user Appdata/Local.
Dbeaver contains the Java 11 JRE needed to run. Verify that the jre/bin folder contains the java files. If not the simpler way would be to reinstall.

If you have java11 installed somewhere else, you can also edit dbeaver.ini and add  (change to the actual location of your installed jre)
```
-vm
C:\Program Files\RedHat\java-11-openjdk-11.0.12-1\bin\javaw.exe
```
Just before the line that says -vmargs
